### PR TITLE
List Training Note

### PR DESCRIPTION
**Pull request by ntqhungdx**
Note list

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import axios from 'axios'
 
-export default {
+export let getNoteList = ( callBack ) => {
```
Comment (furuhiro-jp) [inline src/api/index.js:5]: Is it acceptable to have other APIs defined here as well?

File: src/components/Header.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <header>
+    <div class='header'>
+      <span class='icon-back' @click="handleBack( url )"></span>
+      <span class='title'> {{ title }} </span>
+    </div>
+  </header>
+</template>
```
Comment (furuhiro-jp) [inline src/components/Header.vue:8]: It is a common header. Thank you for creating.

File: src/components/NoteItem.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <section class="item-box item-note-list">
+    <div class="item-box__media">
+      <img v-bind:src="noteItem.thumbUrl" />
```
Comment (furuhiro-jp) [inline src/components/NoteItem.vue]: My component uses abbreviations, is it better to use `v-bind` explicitly?
``

File: src/components/NoteList.vue [hunk 4]
```diff
@@ -1,26 +1,83 @@
 <template>
   <section class="screen-box">
```
Comment (furuhiro-jp) [inline src/components/NoteList.vue:2]: I have a question about HTML coding.
In the parent component, is the following image acceptable?
```
<section class = "screen-box">
   <Header title = "[page title]" url = "[return url]"> </Header>
   <main class = "screen-box__main">
       content
   </main>
</section>
```

**Review by furuhiro-jp — APPROVED** [review]
LGTM.
I asked a few questions such as coding rules.

File: src/api/index.js (hunk 1 above)
Comment (ntq-minhtu) [reply src/api/index.js:5]: Yes, we can define more functions and export it.

Comment (furuhiro-jp) [reply src/api/index.js:5]: I understand. Thank you.

File: src/components/NoteItem.vue (hunk 3 above)
Comment (ntq-minhtu) [reply src/components/NoteItem.vue]: No one is better, but v-bind is verbose, we'll edit.

File: src/components/NoteList.vue (hunk 4 above)
Comment (ntq-minhtu) [reply src/components/NoteList.vue:2]: Don't should have spaces when declare attributes of HTML tags

**Pull request merged**